src/main/java/org/springframework/samples/petclinic/owner/OwnerRepository.java

```diff
@@ -32,6 +32,7 @@
  * @author Sam Brannen
  * @author Michael Isvy
  */
+@org.springframework.stereotype.Repository
```

Comment:
```suggestion
@Repository
```

Reply:
Can't make this change because of compilation error - 'annotation type expected'.